includes/class-wc-stripe-settings.php

```
@@ -0,0 +1,564 @@
+	public function get_skip_pmc_express_checkout_defaults(): string {
+		return $this->settings['skip_pmc_express_checkout_defaults'] ?? 'no';
+	}
+
+	/**
+	 * Get the value of the `stripe_upe_payment_method_order` setting.
+	 *
+	 * @return array<int, string>
+	 */
+	public function get_stripe_upe_payment_method_order(): array {
+		return $this->settings['stripe_upe_payment_method_order'] ?? [];
+	}
```

PHPMD (2.15.0)

[error] The method get_stripe_upe_payment_method_order is not named in camelCase. (undefined)

(CamelCaseMethodName)